parts/k8s/kubernetesagentresourcesvmss.t

```diff
@@ -53,7 +53,7 @@
               "name": "[variables('{{.Name}}VMNamePrefix')]",
               "properties": {
                 "primary": true,
-                "enableAcceleratedNetworking" : "{{.IsAcceleratedNetworkingEnabled}}",
+                "enableAcceleratedNetworking" : "{{.AcceleratedNetworkingEnabled}}",
```

Comment:
See comment below, the function was superfluous, we can just check the bool type instead